### ISSUE TITLE
EOS Error!!!

### ISSUE THREAD

**Issue by Nappster2208** [dliw/fpCEF3#68]
hello...
sorry i'm still a newbie here so i got some trouble installing CEF.
i'm using windows 8.1 x64
lazarus 1.8.4 x64
with cef_binary_ 3.3396.1780
but i got error EOS when tried to run it...

anyone can help me?

**Comment by dliw** [comment]
You have the wrong CEF version. For the latest fpCEF release you have to use CEF 3.3029. Also make sure, that the file structure is correct.

**Comment by thedarkb** [comment]
I'm having this issue too, when do you think newer versions of the CEF will be supported and what needs to be done to add support?

**Comment by dliw** [comment]
Well, I would really like to support the newest CEF version rather sooner than later.

>What needs to be done to add support?  
1. headers need to be updated - this is the easy part and already done
2. the LCL integration needs to be adapted to changes in the behavior of CEF

The second part is the tricky one and that's where I'm stuck. With the newer versions of CEF the current LCL integration no longer works reliably on Linux. I have already tried several ideas but I did not yet find a satisfactory solution. 
Also the current version of fpCEF does not work on OSX (older versions do), which I would like to fix for the next release. This seems to be caused by changes to the event handling, but I did not find the exact cause and don't have a solution yet.

To solve these problems I will have to better understand the inner workings of the LCL and CEF, I guess. However, my time does not permit that at the moment and unfortunately almost certainly also not in the next two months. :disappointed:

**Comment by thedarkb** [comment]
Understandable, have you tried lazwebkit? It lets you link libwebkit in to give you a rendering engine.

**Comment by dliw** [comment]
Yes, I already tried lazwebkit in the past. AFAIK it only supports GTK and has less features than CEF.
Depending on the use case lazwebkit certainly is a suitable way to display HTML pages in an application, but I don't think it helps any further in the scope of fpCEF.

With fpCEF I try to create a cross-platform browser component for Lazarus. However, my focus mainly is on Linux, because on Windows there are other ports of CEF for Delphi or Free Pascal that can be used.
